src/init.cpp

```diff
@@ -965,8 +965,10 @@ bool AppInitParameterInteraction(const ArgsManager& args, bool use_syscall_sandb
     }
 
     peer_connect_timeout = args.GetIntArg("-peertimeout", DEFAULT_PEER_CONNECT_TIMEOUT);
-    if (peer_connect_timeout <= 0) {
+    if (peer_connect_timeout < 0) {
         return InitError(Untranslated("peertimeout cannot be configured with a negative value."));
+    } else if (peer_connect_timeout == 0) {
+        return InitError(Untranslated("invalid value for peertimeout."));
```

Comment:
Reasons for using lowercasel `i` instead of uppercase:

- Error is printed with uppercase `E` in this format: `Error: <error message>`
- Other error starts with lowercase `p` so maintain consistency

Reply:
Other error messages not starting with argument name does not start with lowercase. Lowercase is used above because `peertimeout` and `Peertimeout` will be different arguments. See https://github.com/bitcoin/bitcoin/blob/1642256d2ffb69b3a7386d9f99d584bf8c2df090/src/init.cpp#L1053, for example.